R/pkg/R/DataFrame.R

```diff
@@ -2514,7 +2529,9 @@ setMethod("attach",
 #' environment. Then, the given expression is evaluated in this new
 #' environment.
 #'
+#' @title with
```

Comment:
@shivaram Is this supposed to be a long-form title, or just the name of the method?  Looking at other examples, it looks like it should be a short description